src/messages/MMDSFindIno.h

```diff
@@ -19,8 +19,8 @@
 #include "include/filepath.h"
 
 struct MMDSFindIno : public Message {
-  ceph_tid_t tid;
-  inodeno_t ino;
+  ceph_tid_t tid {0};
+  inodeno_t ino {0};
 
```

Comment:
There is no need of initializing `ino`. See the definition of inodeno_t. By default it is initialized to `0` in the constructor.